src/main/java/com/conveyal/taui/controllers/WorkerController.java

```diff
@@ -77,9 +84,9 @@ public WorkerController (Broker broker) {
      */
     public void register () {
         head("", this::headHandler);
-        get("/internal/jobs", this::getAllJobs);
-        get("/internal/workers", this::getAllWorkers);
         post("/internal/poll", this::workerPoll);
+        get("/api/jobs", this::getAllJobs);
```

Comment:
At this point I think we've only got one endpoint beginning with `/internal`. Probably time to come up with a true strategy for allowing only workers to hit this polling endpoint. We could expose all endpoints publicly and just somehow give the workers credentials to contact this one - not auth0 credentials, but some key only the workers would know but not someone trying to mess with the system. The alternative is to block particular endpoints in the reverse proxy but that is an extra custom setup step that might be nice to avoid if we find a simple way.

Reply:
I'd imagine that would be pretty extensive changes, perhaps would be better to do in a subsequent PR.

Reply:
Definitely, just mentioning it. And I'm not suggesting any specific approach, just saying we need to decide how we'll handle this situation.